all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p>Let's add hyphen to <code>rougel</code> to follow the original library's naming</p><p> </p><p><code>nam</code>  -&gt; <code>name</code></p><p><code>hypt</code>  -&gt; <code>hypothesis</code>  </p><p><code>refr</code>  -&gt; <code>reference</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>